develop/toc.yml

```diff
@@ -5147,6 +5147,8 @@
               topicUid: Microsoft_Visual_Studio_keyboard_shortcuts
             - name: Class Library packages
               topicUid: Class_Library_packages
+    - name: Github CI/CD 
```

Comment:
If this is indeed strictly related to CI/CD, it would make more sense to me to move this to the CI/CD section below this instead of adding it at the same level as that section.

Reply:
Indeed, I'll move it.